repos/system_upgrade/common/actors/initramfs/mounttargetiso/actor.py

```diff
@@ -0,0 +1,16 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import mount_target_iso
+from leapp.models import TargetUserSpaceInfo, TargetOSInstallationImage
+from leapp.tags import IPUWorkflowTag, LateTestsPhaseTag
+
+
+class MountTargetISO(Actor):
+    """Mounts target OS ISO in order to install upgrade packages from it."""
+
+    name = 'mount_target_iso'
+    consumes = (TargetUserSpaceInfo, TargetOSInstallationImage,)
+    produces = ()
+    tags = (LateTestsPhaseTag, IPUWorkflowTag)
```

Comment:
What led you to use this phase and not the "PreparationPhase"? Just asking...

Reply:
The original logic was that the code is testing whether it can mount the target OS ISO, and thus, whether it makes any sense to continue with the upgrade. I have read what the docstring says about the purpose of the LateTests phase and I have moved it to preparation  as suggested. As the fail policies for these two phases are the same it should not make a difference in terms of functionality.